redist/org.goldendict.GoldenDict.metainfo.xml

```diff
@@ -5,6 +5,9 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0-or-later</project_license>
   <name>GoldenDict</name>
+  <developer>
```

Comment:
Looks like the `developer` tag was introduced in AppStream 1.0 released [a few months ago](https://blog.tenstral.net/2023/11/appstream-1-0-released.html). Would the new tag be a problem for distros that package an older version of AppStream? If yes, maybe use the older deprecated `<developer_name> ` tag for the time being?